src/app/components/core/configuration-manager/configuration-manager.component.ts

```diff
@@ -51,24 +78,32 @@ export class ConfigurationManagerComponent implements OnInit {
         });
   }
 
-  public getChildren(category_name, onLoadingPage = false) {
+  public getChildren(category_name, onLoadingPage = false, appendTo = 'x') {
     /** request started */
     this.ngProgress.start();
     this.childCategories = [];
-    this.categoryData = [];
-    this.selectedRootCategory = category_name;
+    if (appendTo === 'x') {
+      this.selectedRootCategory = category_name;
+      this.categoryData = [];
+    }
     this.configService.getChildren(category_name).
       subscribe(
         (data) => {
           /** request completed */
           this.ngProgress.done();
+          console.log('data1111', data['categories']);
           data['categories'].forEach(element => {
-            this.childCategories.push({ key: element.key, description: element.description, is_selected: 'false' });
+            this.childCategories.push({ key: element.key, description: element.description, is_selected: false });
           });
           if (onLoadingPage === true) {
-            this.childCategories[0].is_selected = 'true';
+            this.childCategories[0].is_selected = true;
             this.getCategory(this.childCategories[0].key, this.childCategories[0].description);
           }
+          if (this.childCategories.length) {
+            const h = this.get_child_html(this.childCategories);
+            document.getElementById(appendTo).innerHTML = h;
+          }
+          console.log('childCategories', this.childCategories);
```

Comment:
remove or say `console.log(category_name, ' childCategories: ', this.childCategories);`